### PR TITLE
clean up more resources during factory reset and cleanup script

### LINKED ISSUE #860: Clear support tunnels during cleanup script, factory reset

### PR DESCRIPTION
**Pull request by rtertiaer**
### What does this change intend to accomplish?
This PR fixes #860 , which turned up during the 0.4.2 QA testing. It removes more things during factory reset and from our manufacturing cleanup script.

### Checklist

* [x] Have you tested your changes and ensured they work?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] If applicable, have you updated the documentation/manual?
* [x] If applicable, have you updated the CHANGELOG?
* [x] Does your submission pass linting & tests? You can test on localhost using `./scripts/test` haha CI go brrr
* [ ] Have you written new tests for your core features/changes, as applicable?

### CODE REVIEW

File: amplipi/utils.py [hunk 1]
```diff
@@ -439,3 +439,44 @@ def careful_proc_shutdown(proc: subprocess.Popen, proc_name="process"):
...
+      text=True,
+      check=True
+    )
+    if running_support_tunnels.stdout:
+      for tunnel in [i.split()[0] for i in running_support_tunnels.stdout.split(sep='\n')]:
+        subprocess.run(
+          f"/opt/support_tunnel/venv/bin/python -m invoke stop {tunnel}".split(),
+          cwd="/opt/support_tunnel",
+          check=True
+        )
```
Comment (linknum23) [inline amplipi/utils.py:464]: Should this be wrapped in another try?

File: amplipi/utils.py [hunk 2]
```diff
@@ -439,3 +439,44 @@ def careful_proc_shutdown(proc: subprocess.Popen, proc_name="process"):
...
+
+
+def clear_custom_configs():
+  """ This clears out any custom configs on the appliance for a factory reset, including:
+      * support tunnels
+      * user configs (as of Aug 2024, just an "admin" user with a password set)
+  """
+  # Attempt to clear any support tunnels before wiping configs.
+  try:
+    running_support_tunnels = subprocess.run(
```
Comment (linknum23) [inline amplipi/utils.py:451]: I thought this output json?

Comment (rtertiaer) [reply amplipi/utils.py:451]: nope!
```
(venv) pi@amplipi:/opt/support_tunnel $ inv list-running-tunnels
4096afa4-66b4-4771-8f10-d18ac0747786 10
```

File: amplipi/utils.py (hunk 1 above)
Comment (rtertiaer) [reply amplipi/utils.py:468]: I've done so in my latest commit!

File: amplipi/utils.py [hunk 3]
```diff
@@ -455,25 +456,36 @@ def clear_custom_configs():
...
   try:
-    # Explicitly clear any wireguard configs, in spite of the above cleanup
     wg_configs = subprocess.run("sudo ls /etc/wireguard".split(), capture_output=True, text=True, check=True)
     if wg_configs.stdout:
       for wg_config in wg_configs.stdout.split():
         subprocess.run(f"sudo rm /etc/wireguard/{wg_config}".split(), check=True)
   except Exception as e:
     logger.exception(f"failed to remove wireguard configs: {e}")
 
+  # Remove any user/group that starts with support*, but leave their home directories.
```
Comment (linknum23) [inline amplipi/utils.py:481]: Why leave their home directories?

Comment (linknum23) [reply amplipi/utils.py:481]: ( I would just put that in the comment)

**Review by linknum23 — APPROVED** [review]

**Pull request merged**